lib/tree-view.coffee

```diff
@@ -869,7 +873,7 @@ class TreeView
       return
 
     entryName = path.basename(initialPath)
-    newPath = "#{newDirectoryPath}/#{entryName}".replace(/\s+$/, '')
+    newPath = "#{newDirectoryPath}#{path.sep}#{entryName}".replace(/\s+$/, '')
```

Comment:
Nit: could you remove this change and add it in a separate PR?  It should also be `path.join(newDirectoryPath, entryName)`.

I ask because I'm also fixing this same line in both #1180 and #1173, so it would probably be better to just submit this change in a different PR.